rest/config.go

```diff
@@ -1046,6 +1042,26 @@ func RunServer(config *ServerConfig) {
 	config.Serve(*config.Interface, CreatePublicHandler(sc))
 }
 
+func validateAndAddDBFromConfig(sc *ServerContext, databases DbConfigMap) (warnings []string) {
+	bucketUUIDToDbs := make(map[string][]string, len(databases))
+	for dbName, dbConfig := range databases {
+		dbContext, err := sc.AddDatabaseFromConfig(dbConfig)
+		if err != nil {
+			base.Fatalf("Error opening database %s: %+v", base.MD(dbName), err)
+		}
+		if uuid, err := dbContext.Bucket.UUID(); err == nil {
+			if value := bucketUUIDToDbs[uuid]; value != nil {
+				warningMessage := fmt.Sprintf("Database %q shares bucket %q with databases %s. This may result in unexpected behaviour if security is not defined consistently.",
+					base.MD(dbName), base.MD(dbContext.Bucket.GetName()), base.MD(value))
```

Comment:
Redaction functions inside Sprintfs don't work. The values need to be passed into a log function, or if unavoidable, redacted manually like `base.MD("val").Redact()` - but ...

suggestions further down in the unit test should avoid this by returning values instead of warnings.

Reply:
Since the code has bee revised, this change is obsolete.